src/libraries/Microsoft.Extensions.Http/src/Logging/LoggingHttpMessageHandler.cs

```diff
@@ -43,6 +55,7 @@ public LoggingHttpMessageHandler(ILogger logger, HttpClientFactoryOptions option
             _options = options;
         }
 
+        /// <inheritdoc />
```

Comment:
I'd rather have a full description for this method, especially because this SendAsync has an exception that needs to be explicitly documented.

Reply:
The inherited documentation includes the exception (see below). I could see being more explicit in the summary to include the fact that it is logging and sending the request; but I'm tripping on coming up with how to write that concisely. I've submitted a remark to enhance the inherited documentation.  

If not sufficient please let me know and I'll be glad to copy the inherited documentation forward explicitly.  

//
        // Summary:
        //     Sends an HTTP request to the inner handler to send to the server as an asynchronous
        //     operation.
        //
        // Parameters:
        //   request:
        //     The HTTP request message to send to the server.
        //
        //   cancellationToken:
        //     A cancellation token to cancel operation.
        //
        // Returns:
        //     The task object representing the asynchronous operation.
        //
        // Exceptions:
        //   T:System.ArgumentNullException:
        //     The request was null.

Reply:
We can leave it as is if the description coming from the API this one inherits from, makes sense here too.
Same for the ditto API below.